ha/core/event_manager/subscribe_event.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+class SubscribeEvent:
+  def __init__(self, resource_type : str, states : list):
+    pass
```

Comment:
What are the attributes of the class
self.resource _type?
self.states?

I need to access those in event manager!

Reply:
@indrajitzagade

Reply:
@Madhura-08 Please refer to LLD https://seagate-systems.atlassian.net/wiki/spaces/PUB/pages/499745419/LLD+Event+Manager
Attributes would be e.g. SubscribeEvent('enclosure:cortx:disk_group', ['failed', 'online'])

Reply:
I got it @indrajitzagade. But, I will be receiving the object of this class and to access these things, I need attribute name.
For eg, wvent is an onject of SubscribeEvent, then event.resource_type, event.state will be the way to access it and something similar!

Reply:
yes, I think that is what we need. event.state would be a list though. Please add these two accordingly.

Reply:
I don't see this is added in the new code? Or should I do it in my PR?

Reply:
I think both are already available
 def __init__(self, resource_type : str, states : list):

in system health HEALTH_STATUSES will be status
and events list is added in this pr which are resource type